README.md

```diff
@@ -1,24 +1,84 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
+This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Built With
+- Languages used
+  - Ruby
+  - HTML
+  - CSS
+  - Rails
+  
+- Technologies used
+  - PostgreSql
+  - Rspec
+  - Capybara
 
```

Comment:
- [ ] **Required:** Great job building the app so far, however, some submission requirements are missing:
    - A recorded video to present the project and mention some code you developed that you are very proud of.
    - A link to the deployed version of your application.